mlir/include/mlir/Dialect/Math/Transforms/Passes.h

```diff
@@ -39,6 +42,7 @@ void populateExpandPowFPattern(RewritePatternSet &patterns);
 void populateExpandFPowIPattern(RewritePatternSet &patterns);
 void populateExpandRoundFPattern(RewritePatternSet &patterns);
 void populateExpandRoundEvenPattern(RewritePatternSet &patterns);
+void populateMathExpandPatterns(RewritePatternSet &patterns);
```

Comment:
We don't need to expand every pattern. We want control over what patterns we want to expand and not.